.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor/
-composer.lock
\ No newline at end of file
+composer.lock
+.idea
```

Comment:
`.idea` should be part of your global gitignore rather than the project's

Reply:
I think it's a good idea to keep it in the project's `.gitignore`, so that the project doesn't rely on individual contributors' settings but provides as complete and easy setup as possible for contributing.

Reply:
GitHub themselves recommend putting IDE ignores globally (https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore), since you can't predict all possible editors people might use. If it's desired for a gitignore file to contain IDEs, then we'd probably want to include other IDEs too

Reply:
Yepa, lets pop some more (vscode etc) in there in a separate task